app/src/main/java/org/wikipedia/dataclient/mwapi/MwQueryPage.java

```diff
@@ -177,11 +177,25 @@ public long getLastRevId() {
         @Nullable private String comment;
         @Nullable private Map<String, RevisionSlot> slots;
 
+        @NonNull public String getComment() {
+            return StringUtils.defaultString(comment);
+        }
+
         public long getRevId() {
             return revid;
         }
 
-        @NonNull public String content() {
+        public long getParentRevId() {
+            return parentid;
+        }
+
+        @Nullable
+        public String getUser() {
+            return user;
```

Comment:
Can we follow the pattern of returning a `NonNull` string here? (with `StringUtils.defaultString()`)